app/jobs/regular/create_linked_topic.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module Jobs
+  class CreateLinkedTopic < ::Jobs::Base
+
+    def execute(args)
+      reference_post = Post.find_by(id: args[:post_id])
+      return unless reference_post.present?
+      parent_topic = reference_post.topic
+      return unless parent_topic.present?
+      parent_topic_id = parent_topic.id
+      parent_title = parent_topic.title
+      title_suffix_locale = I18n.t('create_linked_topic.title_suffix')
+
+      linked_topic_record = parent_topic.linked_topic
+      if linked_topic_record.present?
+        raw_title = parent_title.gsub(/\(#{title_suffix_locale} (\d+)\)$/i, "").strip
```

Comment:
Hmm, would this work as well given the other changes I suggest?

```suggestion
        raw_title = parent_title.delete_suffix(I18n.t("create_linked_topic.topic_title_with_sequence", topic_title: "", count: linked_topic_record.sequence))
```

Reply:
Seems like it works!